rollup.config.cjs

```diff
@@ -56,11 +34,11 @@ module.exports = [
   },
 
   // ES6 builds
-  // dist/chart.mjs
+  // dist/chart.js
```

Comment:
I think it would be quite confusing to have dist/chart.js be ESM and dist/chart.min.js UMD. I'd expect chart.min.js to be minified version of chart.js.

Reply:
should we name it `chart.umd.min.js` or just `chart.umd.js` or something?